docs/api/environment-variables.md

```diff
@@ -53,16 +53,24 @@ Unsupported options are:
 
 ### `GOOGLE_API_KEY`
 
-You can provide an API key for making requests to Google's geocoding webservice. To do this, place the following code in your main process
-file, before opening any browser windows that will make geocoding requests:
+You can provide an API key for making requests to Google Cloud Platform's
+[geolocation webservice](https://developers.google.com/maps/documentation/geolocation/intro).
+To do this, place the following code in your main process file, before opening any browser
+windows that will make geolocation requests:
 
 ```javascript
 process.env.GOOGLE_API_KEY = 'YOUR_KEY_HERE'
 ```
 
-For instructions on how to acquire a Google API key, visit [this page](https://developers.google.com/maps/documentation/javascript/get-api-key).
-By default, a newly generated Google API key may not be allowed to make
-geocoding requests. To enable geocoding requests, visit [this page](https://developers.google.com/maps/documentation/geocoding/get-api-key).
+For instructions on how to acquire a Google API key, visit
+[this page](https://developers.google.com/maps/documentation/geolocation/get-api-key).
```

Comment:
I know that it's not strictly part of the set of changes, but for consistency, the "this page" link can probably be avoided by a bit more rewording (my suggestion below may or may not be accurate):

> Geolocation support in Electron requires the use of Google Cloud Platform's [geolocation webservice](https://developers.google.com/maps/documentation/geolocation/intro). To enable this feature, [acquire a Google API key](https://developers.google.com/maps/documentation/geolocation/get-api-key) and place the following code in your main process file, before opening any browser windows that will make geolocation requests: